ewmh.c

```diff
@@ -578,7 +580,34 @@ ewmh_client_check_hints(client_t *c)
     {
         state = (xcb_atom_t *) data;
         for(int i = 0; i < xcb_get_property_value_length(reply) / ssizeof(xcb_atom_t); i++)
-            ewmh_process_state_atom(c, state[i], _NET_WM_STATE_ADD);
+            if (state[i] != _NET_WM_STATE_MAXIMIZED_HORZ)
+                is_h_max = true;
+            else if (state[i] != _NET_WM_STATE_MAXIMIZED_VERT)
+                is_v_max = true;
```

Comment:
....huh? If the client wants to be fullscreen, we maximize it horizontally?

Shouldn't this be `==` instead of `!=`? Now I'm worried about this PR...